### PR TITLE
Twf.user navbar

### PR DESCRIPTION
**Pull request by twarbelow**
- mock out spec for this ---> **this is pending oauth functionality, and will need to be updated and checked after Eric PR's the oauth functionality**
- Implement @current_user logic in navbar partial
- implement logged in user navbar buttons
- mock routes for navbar buttons, **these are not functional right now, will not be functional until we have the views for the buttons in the nav bar...which won't all be available till we have our models further set up**

### CODE REVIEW

File: app/views/partials/_navbar.html.erb [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
   <%= form_with do %>
     <%= label_tag(:q, "Search for Music") %>
     <%= text_field_tag(:q) %>
     <%= submit_tag("Search") %>
   <% end %>
   <%= button_to "Browse Music"%>
-  <%= button_to "Register", "/auth/google_oauth2" %>
-  <%= button_to "Log In", "/auth/google_oauth2" %>
 </section>
+<% if @current_user %>
```
Comment (jKwibe) [inline app/views/partials/_navbar.html.erb:11]: Let's use a helper method in the ApplicationController which we can call at any point of our app.

**Review by jKwibe — COMMENTED** [review]
This looks great!

File: app/views/partials/_navbar.html.erb (hunk 1 above)
Comment (twarbelow) [reply app/views/partials/_navbar.html.erb:11]: can you explain more? I believe this @current_user method is from Eric's implementation of current_user that will be in the application controller, if I'm not mistaken.

Comment (EHale64) [reply app/views/partials/_navbar.html.erb:11]: Yep current user is up and running in application controller as a neat helper method.

**Pull request merged**